src/Controls/tests/TestCases.HostApp/Issues/Issue20166.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
Can you rename this file to Issue20166.xaml.cs?

Reply:
Done